zend_abstract_interface/functions/tests/php8/functions.cc

```diff
@@ -0,0 +1,381 @@
+extern "C" {
```

Comment:
@SammyK I really love this testing infrastructure. You made an awesome job!

Reply:
Thanks! The ZAI SAPI test runner piggybacks off of the original Catch2 work that @morrisonlevi [contributed back in October 2020](https://github.com/DataDog/dd-trace-php/commit/aafae27f64a7475ed77d502f3d06ad0510e40a48). #TeamWork 🙂